config/env.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
```

Comment:
i'm guessing these scripts in `config/` are the result of the ejection and not custom written? do we know that these scripts are necessary to keep/maintain going forward, or is their presence more of a transitional step?

Reply:
Yes, the config files are added to the app directory once the app is ejected. Most of these are for Webpack and Jest. And yes, we'll have to maintain them going forward, which is not as arduous as I thought it would be - the files committed in this PR have already been tweaked slightly to make sure all functionality works.